cdap-client-tests/src/main/java/co/cask/cdap/client/app/AppReturnsArgs.java

```diff
@@ -49,7 +50,8 @@ public void configure() {
     @Path(ENDPOINT)
     @GET
     public void handler(HttpServiceRequest request, HttpServiceResponder responder) {
-      responder.sendJson(200, getContext().getRuntimeArguments());
+      TreeMap<String, String> values = new TreeMap<>(getContext().getRuntimeArguments());
```

Comment:
I think it's because the unit-test itself has wrong assert. You don't need to use TreeMap here, but rather just modify the unit-test assertion to `Assert.assertEquals(propMap, responseMap);`, where the responseMap can simply decoded as `Map<String, String>`.